libs/datamodel/core/tests/validation/composite_types/ignore_field_attribute_is_not_allowed.prisma

```diff
@@ -0,0 +1,23 @@
+datasource mdb {
+  provider = "mongodb"
+  url = env("TESTDBURL")
+}
+
+type A {
+  name String
+  c String @ignore
+}
+
+model B {
+  id Int @id
+  a  A
+}
+
+
+
+// [1;91merror[0m: [1mAttribute not known: "@ignore".[0m
```

Comment:
For later, that error could be nicer. 
Also how specific should the test names be? If it is specific to a database should the db name be in the title?

Reply:
I think we still have problems with how we organize datamodel tests, it's hard to find the right categories. All datamodel tests test exactly one provider (which could be the empty provider) at the moment, but some tests are more general: for example in these tests, we know the validations are on composite types and not mongo-specific. But of course, we currently only have composite types on mongo, so they're all de facto mongo tests. I think a decent criterion is "where does the validation live?". In the connector, or in core but relying on connector capabilities => in a subdirectory with the connector name. In the core => in a more general subdirectory, like "composite_types".

Reply:
Ah and

> For later, that error could be nicer.

Yes, it's difficult. These errors are very readable in test failures because the highlighting is applied, there, but not so much when you read the expectation. I don't know of a solution that would reconcile both, so I went with what we already had.